src/promptflow/tests/executor/e2etests/test_traces.py

```diff
@@ -302,6 +336,31 @@ def assert_otel_traces(self, flow_file):
             output = json.loads(span.attributes["output"])
             assert isinstance(inputs, dict)
             assert output is not None
+        self.validate_openai_tokens(span_list)
+
+    def validate_openai_tokens(self, span_list):
```

Comment:
Add some comments to explain what we are validating.

Reply:
Added.